<?xml version="1.0"?>
<launch>
    <arg name="camera" default="realsense"/>
    <group if="$(eval arg('camera') == 'realsense')">
        <include file="$(find realsense2_camera)/launch/rs_camera.launch">
            <arg name="enable_pointcloud" value="true"/>
        </include>
        <param name="image_topic" value="camera/color/image_raw"/>
        <param name="pointcloud_topic" value="camera/depth/color/points"/>
    </group>
    <group if="$(eval arg('camera') == 'bumblebee')">
        <param name="image_topic" value="/ridgeback/bumblebee/left/image/color"/>
        <param name="pointcloud_topic" value="/ridgeback/bumblebee/points2"/>
    </group>
    <!-- <remap from="camera/color/image_raw" to="camera/rgb/image_raw"/> -->
    <node pkg="object_detection" type="object_detection.py" name="object_detection" output="screen" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find object_detection)/rviz/object_detection.rviz"/>
</launch>